lib/widgets/main_screen_bottomnavbar.dart

```diff
@@ -0,0 +1,132 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:provider/provider.dart';
+
+import '../change_notifiers/tab_manager.dart';
+
+class MainScreenBottomNavBar extends StatelessWidget {
+  final Function onTap;
+
+  const MainScreenBottomNavBar({super.key, required this.onTap});
+
+  @override
+  Widget build(BuildContext context) {
+    const double borderRadius = 30.5;
+
+    return Container(
+      decoration: BoxDecoration(
+        color: Theme.of(context).colorScheme.tertiary,
+        borderRadius: const BorderRadius.only(
+          topLeft: Radius.circular(borderRadius),
+          topRight: Radius.circular(borderRadius),
+        ),
+      ),
+      height: 70,
+      padding: const EdgeInsets.only(
+        top: 10,
+        left: 10,
+        right: 10,
+        bottom: 0,
+      ),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+        children: [
+          _SelectedIcon(
```

Comment:
create a model where it will accept image and asset or a collection then map or for loop it here, it here and returns _SelectedIcon

```dart
someCollection.map((s) => _SelectedIcon(// put properties)).toList()
```